<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Durak Listesi</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>22</y>
         <width>571</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="StationName">
            <property name="placeholderText">
             <string>Durak Adı</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_add">
            <property name="text">
             <string>Ekle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_remove">
            <property name="text">
             <string>Çıkar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Modyo</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>31</y>
         <width>215</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="button_send_list">
          <property name="text">
           <string>Durak Lİstesini yolla</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_giris">
            <property name="text">
             <string>Giriş</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_cikis">
            <property name="text">
             <string>Çıkış</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button_endofline">
          <property name="text">
           <string>Hat Sonu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
